src/searchengine/nova/helpers.py

```diff
@@ -35,12 +35,16 @@
 import re
 import socket
 import socks
+import ssl
 import StringIO
 import tempfile
 import urllib2
 
+# Some sites have invalid certificates
+ssl._create_default_https_context = ssl._create_unverified_context
```

Comment:
I think this should be done case by case, I mean to the respective plugin instead, not here.

Reply:
I agree, but I don't have time to keep all plugins updated with minor things like that. In most sites, certificates's issues are temporary until the admin fix it. I don't think this represents a very serious security problem.
Please, don't block the PR :/

Reply:
This definitely should not be done for all torrent sites. This needs to be an exception to the rule, and not the rule itself.